convert-to-declarative-plugin/src/main/webapp/css/todeclarative.css

```diff
@@ -304,7 +304,7 @@
     background-color: #4B758B;
     display: table-cell;
     margin-top: 10px;
-    //margin-left: 400px;
+    /*margin-left: 400px;*/
```

Comment:
CSS doesn't support `//` comments. Not sure if it actually mattered in practice, just triggered a warning in my IDE, and the page appears to look the same with or without the change.